gradio_tts_app.py

```
@@ -272,103 +292,111 @@ def load_and_prepare_model():
-    Parameters:
-        text (str): The input text to synthesize.
-        audio_prompt_path (str): Path to a reference audio file for voice conditioning, or None to use the default voice.
-        exaggeration (float): Controls the expressiveness of the generated speech.
-        temperature (float): Sampling temperature for generation randomness.
-        seed_num (int): Random seed for reproducibility; 0 disables seeding.
-        cfgw (float): Classifier-free guidance weight.
-        min_p (float): Minimum probability threshold for nucleus sampling.
-        top_p (float): Top-p (nucleus) sampling parameter.
-        repetition_penalty (float): Penalty to discourage repetition in output.
-        steps (int): Maximum number of generation steps.
+def generate(text, ref_wav, exaggeration, cfg_weight, steps, seed_num, temp, min_p, top_p, repetition_penalty, enable_chunking, chunk_size):
```

Pylint (3.3.7)

[refactor] Too many arguments (12/5)

(R0913)